its/sources_pr_analysis/changed/Budget.php

```diff
@@ -7,7 +7,6 @@
 
 class Budget extends AbstractController
 {
-
   public function index(Response $response) {
```

Comment:
I will suggest to add new method here to have more practical real-world behavior. Once when I compared this file with base I missed this little change. And new method will create CPD tokens on different lines.